src/main/kotlin/com/ampnet/blockchainapiservice/model/result/FullErc20BalanceRequest.kt

```diff
@@ -0,0 +1,48 @@
+package com.ampnet.blockchainapiservice.model.result
+
+import com.ampnet.blockchainapiservice.model.ScreenConfig
+import com.ampnet.blockchainapiservice.util.BlockNumber
+import com.ampnet.blockchainapiservice.util.ChainId
+import com.ampnet.blockchainapiservice.util.ContractAddress
+import com.ampnet.blockchainapiservice.util.Erc20Balance
+import com.ampnet.blockchainapiservice.util.SignedMessage
+import com.ampnet.blockchainapiservice.util.Status
+import com.ampnet.blockchainapiservice.util.WalletAddress
+import com.fasterxml.jackson.databind.JsonNode
+import java.util.UUID
+
+data class FullErc20BalanceRequest(
+    val id: UUID,
+    val status: Status,
+    val chainId: ChainId,
+    val redirectUrl: String,
+    val tokenAddress: ContractAddress,
+    val blockNumber: BlockNumber?,
+    val walletAddress: WalletAddress?,
+    val arbitraryData: JsonNode?,
+    val screenConfig: ScreenConfig,
```

Comment:
Shouldn't `screenConfig` be optional?
https://www.notion.so/ampnet/Balance-checker-flow-bef6c7d8c85c446c8cf0b335d31461ea#09bf4cdeff204ce5823f47e09151c672

Reply:
Yes, and it will be optional in API request and response. In the service layer `ScreenConfig` is kept as non-null for simplicity. Have a look at `ScreenConfig.orEmpty()` and how it was done for send requests.

Reply:
Nice!